plugin/dashboard.lua

```diff
@@ -1,14 +1,22 @@
 -- version 0.2.3
-if vim.g.loaded_dashboard then
-  return
-end
 
-vim.g.loaded_dashboard = 1
+vim.api.nvim_create_augroup('dashboard', { clear = true })
```

Comment:
there will return id for group

Reply:
Should we save and use the group id? I use `group` name [here](https://github.com/nvimdev/dashboard-nvim/pull/450/files#diff-3fd880e15253451a204b448a20fc498b4cfc6e9ace903c3dfaefb8bd29dfea1dR6) instead of id.

Reply:
Updated, please kindly check